utils/rpms/daos.spec

```diff
@@ -242,8 +242,8 @@ mkdir -p %{?buildroot}/%{conf_dir}/certs/clients
 mv %{?buildroot}/%{_prefix}/etc/bash_completion.d %{?buildroot}/%{_sysconfdir}
 
 %pre server
-getent group daos_admins >/dev/null || groupadd -r daos_admins
-getent passwd daos_server >/dev/null || useradd -s /sbin/nologin -r daos_server
+getent group daos_server >/dev/null || groupadd -r daos_server
```

Comment:
As we discussed, I don't think this is strictly necessary in that the `useradd` utility will create a group if one is not specified. That having been said, it doesn't seem harmful to be explicit.

Reply:
The catch is that you'll need to test for existence of the group anyway: If the group **does** exist and you run useradd **without** "-g", it fails (I just caused that bug, and fixed it in another commit below). If the group does ***not*** exist and you run useradd **with** "-g", that fails. So I decided to always test||create the group, so I am sure that the following "useradd -g" always works.